Deprecating v1.3 from docs site

matthewshaver opened the pull request:
## What are you changing in this pull request and why?

Removes v1.3 from the dropdown
Removes all lastversion="1.3" data
Removes version blocks from firstversion="1.3" data
Alter some sentences that refer to v1.3

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

matthewshaver committed:
Deprecating v1.3 from docs site

matthewshaver committed:
fixing link

matthewshaver committed:
Testing link checker

matthewshaver committed:
Merge branch 'current' into 1dot3-deprecation

matthewshaver committed:
Removing test link

matthewshaver committed:
Merge branch '1dot3-deprecation' of https://github.com/dbt-labs/docs.getdbt.com into 1dot3-deprecation

matthewshaver committed:
Deprecating v1.4 from the docs

nghi-ly reviewed (approved):
looks good! have question related to upgrade guides (below). approving!

when does it make sense to move the 1.3 upgrade guide to the Older folder? is it still needed for v1.0 users who want to upgrade to the latest supported (ie 1.6)?

matthewshaver committed:
Update website/docs/reference/resource-configs/snowflake-configs.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

matthewshaver committed:
Update events-logging.md

Removing calliout

matthewshaver committed:
1.4 deprecation from docs (#5597)

## What are you changing in this pull request and why?

Removes v1.4 from the version dropdown
Removes info between lastVersion="1.4" blocks
Removes firstVersion="1.4" blocks but leaves content
Removes some mentions of v1.4 from places where it's not mentioning a
requirement

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] For [docs
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning),
review how to [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages

matthewshaver committed:
Merge branch 'current' into 1dot3-deprecation
